src/index.tsx

```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
+if (process.env.REACT_APP_MAINNET_ENV === 'production') {
```

Comment:
change this base on domain, because auto deployment domain also use production env

Reply:
For testing, we can use this for current adpr domain, but before merge, please change it to window.location.href.include("kyberswap.com")

Reply:
Why is `window.location.href.include("kyberswap.com")`, I though it was `dmm-interface-pr`.

Reply:
I mean before you merge this PR to master, change it to kyberswap.com
And now for testing, this is ok

Reply:
@nguyenhuudungz : pls resolve this issue